clusterloader2/testing/load/dns-deployment.yaml

```diff
@@ -0,0 +1,61 @@
+{{$HostNetworkMode := DefaultParam .CL2_USE_HOST_NETWORK_PODS false}}
+{{$CpuRequest := DefaultParam .CpuRequest "10m"}}
+{{$MemoryRequest := DefaultParam .MemoryRequest "10M"}}
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Name}}
+  labels:
+    group: load
+    svc: {{.SvcName}}-{{.Index}}
+spec:
+  replicas: {{RandIntRange .ReplicasMin .ReplicasMax}}
+  selector:
+    matchLabels:
+      name: {{.Name}}
+  template:
+    metadata:
+      labels:
+        group: load
+        name: {{.Name}}
+        svc: {{.SvcName}}-{{.Index}}
+    spec:
+      hostNetwork: {{$HostNetworkMode}}
+      containers:
+      - image: gcr.io/pavithrar-k8s-dev/dnstest:latest
```

Comment:
If we decide to go with this one - I would like the source code of this to be merged in perf-tests repo too.

Reply:
Changed the deployment file to use dnsperf which already has the source code in the perf-tests repo.